aws-datastore/src/androidTest/java/com/amplifyframework/datastore/BasicCloudSyncInstrumentationTest.java

```diff
@@ -103,17 +101,31 @@ public void setup() throws AmplifyException {
         api = SynchronousApi.delegatingTo(apiCategory);
         appSync = SynchronousAppSync.using(AppSyncClient.via(apiCategory));
 
+        long tenMinutesAgo = new Date().getTime() - TimeUnit.MINUTES.toMillis(10);
+        Temporal.DateTime tenMinutesAgoDateTime = new Temporal.DateTime(new Date(tenMinutesAgo), 0);
         DataStoreCategory dataStoreCategory = DataStoreCategoryConfigurator.begin()
             .api(apiCategory)
-            .clearDatabase(true)
```

Comment:
This deletes the database file before each test run.  This was causing the second test to fail each time, and I'm not entirely sure why.  Instead, I am now calling datastore.clear() after each test, in the `teardown()` method.